lib/vf_analyzers.cpp

```diff
@@ -715,7 +715,8 @@ struct ValueFlowAnalyzer : Analyzer {
                 return {value->intvalue == 0};
             ProgramMemory pm = pms.get(tok, ctx, getProgramState());
             MathLib::bigint out = 0;
-            if (pm.getContainerEmptyValue(tok->exprId(), out))
+            // TODO: do we really went to return an impossible value?
+            if (pm.getContainerEmptyValue(tok->exprId(), out, true))
```

Comment:
It is the correct behaviour, but the parameter and the comment makes it look like its wrong. The `getContainerEmptyValue` returns true or false if the container is empty or not. It will use impossible values internally to determine this as well(its really an implementation detail though). 

This is why the `impossible` parameter should not be added.